src/ServiceInsight/Search/SearchBarViewModel.cs

```diff
@@ -52,22 +51,22 @@ protected override void OnActivate()
 
         public void GoToFirstPage()
         {
-            Parent.RefreshMessages(SelectedEndpoint, 1, SearchQuery);
+            Parent.RefreshMessages(FirstLink, 1);
```

Comment:
We need to display the page index in the UI. The current SC unfortunately does not include the current page index in the response but will include in future. Then we'll able to drop these.